themes/default/content/registry/packages/azuread/installation-configuration.md

```diff
@@ -5,6 +5,10 @@ layout: installation
 ---
 
 The Pulumi AzureAD provider uses the AzureAD SDK to manage and provision resources.
+To provision resources with the Pulumi AzureAD provider, you need to have the Azure CLI installed and you need to have Azure credentials.
```

Comment:
You don't need the CLI for service principal authentication

Reply:
ack! removed